src/pages/manager/AddCustomerPage.js

```diff
@@ -3,9 +3,64 @@ const { expect } = require('@playwright/test');
 export class AddCustomerPage {
   constructor(page) {
     this.page = page; 
-  }
+    this.firstNameInput = page.getByPlaceholder('First Name');
+    this.lastNameInput = page.getByPlaceholder('Last Name');
+    this.postCodeInput = page.getByPlaceholder('Post Code');
+    this.addCustomerButton = page. getByRole('form').getByRole('button', { name: 'Add Customer' });
```

Comment:
There is an extra space between `page.` and `getByRole('form')`. Consider removing the space to maintain consistent formatting.